snowexsql/tables/layer_data.py

```diff
@@ -1,6 +1,25 @@
-from sqlalchemy import Column, Float, String
+from sqlalchemy import Column, Float, Integer, String, ForeignKey
+from sqlalchemy.orm import Mapped
+from sqlalchemy.orm import mapped_column
+from typing import List
+from sqlalchemy.orm import relationship
 
 from .base import Base, Measurement, SingleLocationData
+from .observers import Observer
+from .instrument import Instrument
+from .site import Site
+
+
+class LayerObservers(Base):
+    """
+    Link table
+    """
+    __tablename__ = 'layer_observers'
+    __table_args__ = {'schema': 'public'}
+
+    id = Column(Integer, primary_key=True)
```

Comment:
Link tables don't need a primary key.

See [this example](https://www.slingacademy.com/article/sqlalchemy-how-to-use-many-to-many-relationships/) for reference